src/main/java/games/coltexpress/actions/roundcardevents/EndCardHostage.java

```diff
@@ -23,7 +23,6 @@ public boolean execute(AbstractGameState gs) {
         for (Integer playerID : locomotive.playersInsideCompartment){
             gameState.addLoot(playerID, new Loot(Purse, reward));
         }
-        gameState.setGameStatus(Utils.GameResult.GAME_END);
```

Comment:
Lost in the changes, is this set/checked somewhere else?

Reply:
Yes - that is now done in endGame()